src/main/java/uk/gov/justice/probation/courtcaseservice/restclient/communityapi/model/CommunityApiAppointment.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.justice.probation.courtcaseservice.restclient.communityapi.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+
+@Getter
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class CommunityApiAppointment {
```

Comment:
Maybe rename this? `CommunityApiAttendance` or something like that might be clearer?  Up to you

Reply:
Sure makes more sense, I'll update it now.

Reply:
Actually, just had a look and we already have `CommunityApiAttendance` so I'll keep it as `CommunityApiAppointment`, it's actually referred to as `appointment` in community-api.